pkg/crc/machine/machine.go

```
@@ -0,0 +1,177 @@
+package machine
+
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/errors"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/crc/output"
+
+	"github.com/code-ready/crc/pkg/crc/machine/config"
+	"github.com/code-ready/crc/pkg/crc/machine/libvirt"
+
+	"github.com/code-ready/machine/libmachine"
+	"github.com/code-ready/machine/libmachine/log"
+	"github.com/code-ready/machine/libmachine/state"
+	"github.com/code-ready/machine/libmachine/host"
+)
+
+func init() {
+}
+
+func Start(startConfig StartConfig) (StartResult, error) {
+	result := &StartResult{Name: startConfig.Name}
+
+	// Set libmachine logging
+	setMachineLogging(startConfig.Debug)
+
+	libMachineAPIClient := libmachine.NewClient(constants.MachineBaseDir, constants.MachineCertsDir)
+	defer libMachineAPIClient.Close()
+
+	machineConfig := config.MachineConfig{
+		Name:       startConfig.Name,
+		BundlePath: startConfig.BundlePath,
+		VMDriver:   startConfig.VMDriver,
+		CPUs:	    2,
+		Memory:     8096,
+	}
+
+	exists, err := existVM(libMachineAPIClient, machineConfig);
+	if !exists {
+		output.Out("Creating VM")
+
+		host, _ := createHost(libMachineAPIClient, machineConfig)
```

Any reason we are ignoring the error here?